TsRandomizer/Randomisation/BestiaryManager.cs

```diff
@@ -642,8 +663,8 @@ public static void UpdateBestiary(Level level, SettingCollection gameSettings)
 				}
 				if (gameSettings.BossRando.Value != "Off" && validBosses.Contains(bestiaryEntry.Index))
 				{
-					BossAttributes replacedBossInfo = GetBossAttributes(level, bestiaryEntry.Index);
-					BossAttributes vanillaBossInfo = GetVanillaBoss(level, bestiaryEntry.Index);
+					BossAttributes replacedBossInfo = GetReplacedBoss(level, bestiaryEntry.Index);
```

Comment:
I realize now that this likely broke boss scaling, so that will need to be fixed

Reply:
fixed in 835adb3